src/main/java/com/epam/team1/emotional/help/controller/UserController.java

```diff
@@ -20,7 +21,7 @@
 public class UserController {
 
     private final UserService userService;
-    @Autowired
+
     private QuizHistoryService quizHistoryService;
```

Comment:
It's should not work because `QuizHistoryService` isn't final. From the lombok Javadoc:
_public @interface RequiredArgsConstructor
Generates a constructor with required arguments. Required arguments are final fields and fields with constraints such as @NonNull._

Reply:
Aaa that is why it sometimes does not work